### PR TITLE
MAINT: Remove combineAdd and combineMult

### PR DESCRIPTION
**Pull request by gfyoung**
Deprecated in 0.17.0.

xref #10735

### CODE REVIEW

File: doc/source/whatsnew/v0.20.0.txt [hunk 1]
```diff
@@ -815,6 +815,8 @@ Removal of prior version deprecations/changes
 - ``Series``, ``Index``, and ``DataFrame`` have dropped the ``sort`` and ``order`` methods (:issue:`10726`)
 - Where clauses in ``pytables`` are only accepted as strings and expressions types and not other data-types (:issue:`12027`)
 - The ``LongPanel`` and ``WidePanel`` classes have been removed (:issue:`10892`)
+- ``DataFrame`` has dropped the ``combineAdd`` and ``combineMult`` methods in favor of ``add`` and ``mul`` respectively (:issue:`10735`)
+- ``Series``, ``Index``, and ``DataFrame`` have dropped the ``convert_objects`` method (:issue:`11173`)
```
Comment (jreback) [inline doc/source/whatsnew/v0.20.0.txt]: revert this line :<

Comment (gfyoung) [inline doc/source/whatsnew/v0.20.0.txt]: Yep, just needed to rebase onto `master`.  Fixed.

Comment (gfyoung) [inline doc/source/whatsnew/v0.20.0.txt]: Yep, done

**Comment by jreback** [comment]
thanks!

**Pull request merged**